docker-compose.yml

```diff
@@ -194,6 +194,30 @@ services:
       - 'openmrsdbdata:/var/lib/mysql'
     logging: *log-config
 
+  metabasedb:
+    image: postgres:${METABASE_POSTGRES_IMAGE_TAG:?}
+    restart: on-failure:[3]
+    profiles: ["metabase","default"]
+    environment:
+      POSTGRES_DB: ${METABASE_DB_NAME:?}
+      POSTGRES_USER: ${METABASE_DB_USER:?}
+      POSTGRES_PASSWORD: ${METABASE_DB_PASSWORD:?}
+    volumes:
+      - 'metabase-data:/var/lib/postgresql/data'
+    ports:
+     - 5432:5432
```

Comment:
Aren't these ports being used by postgresDB of Odoo already in this compose file?

Reply:
Yes Gurpreet, we are using 5431 for odoodb and 5432 for openelisdb. Shall we use 5430 for metabasedb then?

Reply:
Yes, as long as you choose a unique one, it should be fine. 5430 or 5433. Plz validate once with docker-compose file to ensure its unqiue.